gammapy/catalog/fermi.py

```diff
@@ -319,6 +306,24 @@ def spectral_model(self):
         return model
 
     @property
+    def _flux_points_table_formatted(self):
+        """Returns formatted version of self.flux_points.table"""
+        table = self.flux_points.table.copy()
```

Comment:
I'd suggest to shorten this method a bit:
- remove all empty lines (empty lines are only useful if they separate steps, just an empty line between every line of code is uncommon)
- Break the line with the long list earlier
- Use a list comprehension for
```
[table[_].format = '.3' for _ in flux_cols]
```
because it's just as readable but one line shorter.